velox/core/PlanNode.h

```diff
@@ -14,13 +14,14 @@
  * limitations under the License.
  */
 #pragma once
+
 #include <fmt/format.h>
+
 #include "velox/connectors/Connector.h"
 #include "velox/core/Expressions.h"
 #include "velox/core/QueryConfig.h"
 
-#include "velox/vector/arrow/Abi.h"
-#include "velox/vector/arrow/Bridge.h"
+struct ArrowArrayStream;
```

Comment:
Would it make sense to add 'extern' here ?

Reply:
No need. `extern` is primarily for global variables. We are forward declaring a struct here.